tests/modules/test_measureobjectskeleton.py

```diff
@@ -1,4 +1,4 @@
-'''test_measureneurons.py - test the MeasureNeurons module
+ObjectSkeleton'''test_measureobjectskeleton.py - test the MeasureObjectSkeleton module
```

Comment:
Remove this first `ObjectSkeleton` before the triple quotes- it's keeping it from allowing the test to run.